common/common.scss

```diff
@@ -387,3 +387,19 @@ summary.select-kit-header.single-select-header.dropdown-select-box-header {
     border: 1px dashed var(--tertiary);
   }
 }
+
+// Control order of plugin connectors
+.search-banner {
+  display: none;
+}
+
+.showcased-topic-list {
+  display: none;
```

Comment:
This is a little sub-optimal, because the component would be rendered twice, correct? We're hiding one of the two instances. I guess there's no way around it?

Reply:
Yeah unfortunately, I can't really see another way around it... 😞